components/brave_wallet_ui/page/screens/composer_ui/select_token_modal/select_token_modal.tsx

```diff
@@ -167,270 +235,296 @@ export const SelectTokenModal = React.forwardRef<HTMLDivElement, Props>(
       })
     })
 
-    const { userVisibleTokensInfo } = useGetUserTokensRegistryQuery(undefined, {
-      selectFromResult: (result) => ({
-        userVisibleTokensInfo: selectAllVisibleUserAssetsFromQueryResult(result)
-      })
-    })
+    const { data: combinedTokenRegistry } = useGetCombinedTokensRegistryQuery()
 
-    const { data: fullTokenList } = useGetCombinedTokensListQuery(
-      showFullFlatTokenList ? undefined : skipToken
-    )
+    const fullVisibleTokenList = React.useMemo(() => {
+      if (!combinedTokenRegistry) {
+        return []
+      }
+      return getEntitiesListFromEntityState(
+        combinedTokenRegistry,
+        combinedTokenRegistry.visibleTokenIds
+      )
+    }, [combinedTokenRegistry])
```

Comment:
Looks like we are only using this list for fungible tokens in `tokensBySelectedComposerOption`
We should use `fungibleVisibleTokenIds` instead in that case.

```suggestion
    const fullVisibleFungibleTokensList = React.useMemo(() => {
      if (!combinedTokenRegistry) {
        return []
      }
      return getEntitiesListFromEntityState(
        combinedTokenRegistry,
        combinedTokenRegistry.fungibleVisibleTokenIds
      )
    }, [combinedTokenRegistry])
```

Reply:
Updated to use `fungibleVisibleTokenIds`